frontend/ui/widget/virtualkeyboard.lua

```diff
@@ -343,12 +346,15 @@ function VirtualKey:onTapSelect(skip_flash)
     -- just in case it's not flipped to false on hold release where it's supposed to
     self.keyboard.ignore_first_hold_release = false
     if self.flash_keyboard and not skip_flash and not self.skiptap then
-        self[1].inner_bordersize = self.focused_bordersize
```

Comment:
No more inner border stuff anywhere ? What's flashing then ?

You're also disabling that here in onTapSelect and onSwipeSelect below ?

Reply:
It's handled in the `invert` method ;).